projects/lights_control/inc/lights_config.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "gpio.h"
+
+typedef struct LightsConfig {
+  const GPIOAddress *board_type_address;
+  const GPIOAddress *addresses_front;
+  const GPIOAddress *addresses_rear;
+  uint8_t num_addresses_front;
+  uint8_t num_addresses_rear;
+  const GPIOSettings *gpio_settings_in;
+  const GPIOSettings *gpio_settings_out;
+} LightsConfig;
+
+typedef enum {
```

Comment:
Please fix your naming so it meets our standards (module name first). This goes for other modules as well.

Reply:
ah ok.